### ISSUE TITLE
nvim-tree icons not showing properly

### ISSUE THREAD

**Issue by nanivignesh** [siduck76/NvChad#2286]
TLDR:this is my first time filing a github issue excuse me if any wrong

The bug iam facing is the icons are not showing properly in nvim-tree when a folder is in closed state but the icons are showing when i open the folder that is on typing enter and also did :NvChadUpdate

**To Reproduce**
1. Go to any folder and press <leader+e>
Look at the icons of the file when a folder is in closed state
**Expected behavior**
The default file icons should show properly

file state as close
![image](https://github.com/NvChad/NvChad/assets/100442198/6eb28c7a-e0db-4ee8-bfb6-100e9c760caf)

file state as open
![image](https://github.com/NvChad/NvChad/assets/100442198/a7d476f0-0159-477b-916d-97f06f8c0762)

here is plugins/config/nvimtree.lua
![image](https://github.com/NvChad/NvChad/assets/100442198/6e9950f3-9650-4cd9-b6a8-50e259f51d82)

 - Operating System
 WSL2 (Ubunti 22.0.4.2 LTS)
 - Terminal
 zsh
 - Version of Neovim
0.9

**Comment by siduck** [comment]
pls update nerdfonts version
